app/assets/javascripts/pageflow/editor/templates/file_item.jst.ejs

```diff
@@ -16,6 +16,16 @@
 
   <div class="file_meta_data">
     <table cellpadding="0" cellspacing="0">
+      <tbody>
+        <tr>
+          <th><%= I18n.t('pageflow.editor.templates.file_item.credits') %></th>
+          <td><input type="text" class="rights"/></td>
+        </tr>
+        <tr>
+          <th><%= I18n.t('pageflow.editor.templates.file_item.alt_text') %></th>
+          <td><input type="text" class="alt_text"/></td>
+        </tr>
+      </tbody>
```

Comment:
When you created the mock up, we decided to keep inputs out of the file info area. This can be added to the edit files view instead.